modules/cart/commerce_cart.module

```diff
@@ -223,3 +247,19 @@ function commerce_cart_views_data_alter(array &$data) {
 function template_preprocess_commerce_cart_block(&$variables) {
   $variables['attributes']['class'][] = 'cart--cart-block';
 }
+
+/**
+ * Implements hook_cron().
+ */
+function commerce_cart_cron() {
+  $queue = \Drupal::queue('cart_expirations');
+
+  $ids = \Drupal::entityQuery('commerce_order')
+    ->condition('cart', TRUE)
+    ->execute();
+
+  // Queue ids in groups of 50.
+  for ($i = 0; $i < count($ids); $i += 50) {
```

Comment:
At first, I didn't get this, but now I do. You're putting groups of 50 cart orders into multiple queue items.  

Perhaps we should just put `->range(0, 50)` on the query and just purge 50 at a time. Out of the box, Drupal's automated_cron should run every three hours. I'm sure most people configure it to run more via a command line operation.

@bojanz thoughts? should we just create one queue item for up to 50 orders per run?